packages/asciidoctor-revealjs/package.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
You should probably add `"private": true` and keep it to a minimum with only `bin`, `dependencies` and `devDependencies`.
You don't need all the project information since it won't be published.